tests/console/docker.pm

```diff
@@ -100,12 +100,18 @@ sub run {
     die("error: missing container $container_name") unless ($output_containers =~ m/$container_name/);
 
     # containers state can be saved to a docker image
-    my $output = script_output("docker container exec $container_name zypper -n in curl", 300);
-    die('error: curl not installed in the container') unless ($output =~ m/Installing: curl.*done/);
+    # note that here script_run do not return in case of failure,
+    # so returned value is more an exit_code than an output string.
```

Comment:
Not sure what you are referring to with your comment. The return value of `script_run` is not `more an exit_code` but is just that, an exit code :)

Reply:
comment removed.